src/components/Wizard/Stories/WizardPatternExample.js

```diff
@@ -0,0 +1,197 @@
+import PropTypes from 'prop-types';
+import { boolean, number } from '@storybook/addon-knobs';
+import { Button } from '../../Button';
+import { bindMethods, Wizard } from '../../../index';
+
+const wizardSteps = [
+  {
+    title: 'General',
+    render: () => <p>(Step 1 Contents Here)</p>
+  },
+  {
+    title: 'Details',
+    render: () => <p>(Step 2 Contents Here)</p>
+  },
+  {
+    title: 'Results',
+    render: () => <p>(Step 3 Contents Here)</p>
+  }
+];
+
+export class WizardPatternExample extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      showModal: false,
+      loading: false,
+      activeStepIndex: 0 // eslint-disable-line no-unused-state
+    };
+    bindMethods(this, ['close', 'open', 'renderStateless', 'renderStateful']);
+  }
+  close() {
+    this.setState({ showModal: false, loading: false });
+  }
+  open() {
+    this.setState({ showModal: true, loading: true });
+    setTimeout(() => {
+      this.setState({ loading: false });
+    }, 1000);
+  }
+  renderStateless() {
+    // We need knobs on the stateless example, because we must drive its state ourselves.
+    const { loading, activeStepIndex } = this.state;
+
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+          Launch Stateless Wizard
+        </Button>
+
+        <Wizard.Pattern
+          show={this.state.showModal}
+          onHide={this.close}
+          onExited={this.close}
+          title="Stateless Wizard Pattern Example"
+          nextStepDisabled={false}
+          steps={wizardSteps}
+          loadingTitle="Loading..."
+          loadingMessage="This may take a minute."
+          onStepChanged={index => this.setState({ activeStepIndex: index })}
+          loading={boolean('Loading', loading)}
+          activeStepIndex={number('Active Step Index', activeStepIndex, {
+            range: true,
+            min: 0,
+            max: 2
+          })}
+        />
+      </div>
+    );
+  }
+  renderStateful() {
+    // No knobs for the stateful example, we want to let it control its own state.
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+          Launch Stateful Wizard
+        </Button>
+
+        <Wizard.Pattern.Stateful
+          show={this.state.showModal}
+          onHide={this.close}
+          onExited={this.close}
+          title="Stateful Wizard Pattern Example"
+          shouldDisableNextStep={activeStepIndex => false}
+          steps={wizardSteps}
+          loadingTitle="Loading..."
+          loadingMessage="This may take a minute."
+          loading={this.state.loading}
+        />
+      </div>
+    );
+  }
+  render() {
+    return this.props.stateful ? this.renderStateful() : this.renderStateless();
+  }
+}
+
+WizardPatternExample.propTypes = {
+  stateful: PropTypes.bool
+};
+
+WizardPatternExample.defaultProps = {
+  stateful: false
+};
+
+export const wizardPatternExampleSource = `
```

Comment:
this seems excessive/impossible to maintain.  Is there another way this could be acquired, or could it be toned down to only the bare minimum?

Reply:
welcome to party @dmiller9911 🕺 !!

i agree w/ yah completely. 

do you mind opening an issue? It would be great to clean these up across the board in a future PR. I started to look at [jsx-to-string](https://www.npmjs.com/package/jsx-to-string) a bit back but never made it through. Maybe you can find a better approach? Only the "stateful" story examples have this issue...but would be nice to kill off that convention in the future...

Reply:
I ran into this issue as well. I think it has to do something with Storybook `withInfo` addon not being able to resolve the React component source properly, since there's `source: false` all across the stories. So people end up with passing

```
text: (
  <div>
    <h1>Story Source</h1>
    <pre>{mockLoadingWizardSource}</pre>
  </div>
)
```

to the `withInfo` addon, which beats the purpose of its automatic source inclusion feature.